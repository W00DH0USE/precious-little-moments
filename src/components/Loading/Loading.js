import React from 'react';
import Spinner from 'react-spinner-material';
import './Loading.css';

function Loading() {
  return (
    <div className="loading-container">
      <Spinner size={120} spinnerColor={"#FFF"} spinnerWidth={2} visible={true} />
    </div>
  )
}

export default Loading;