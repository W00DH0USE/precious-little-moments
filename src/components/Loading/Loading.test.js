import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loading from './Loading';


describe(`Loading component`, () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<Loading />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});