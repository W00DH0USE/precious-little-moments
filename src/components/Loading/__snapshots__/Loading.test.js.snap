// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders without crashing 1`] = `
<div
  className="loading-container"
>
  <Spinner
    size={120}
    spinnerColor="#FFF"
    spinnerWidth={2}
    visible={true}
  />
</div>
`;
